docs/tutorial/sql.md

```diff
@@ -509,26 +509,104 @@ There are lots of other functions can be combined with these queries. Please rea
 
 ==Sedona >= 1.5.0==
```

Comment:
No need to add Sedona >= 1.5.0 here. Sedona website has a version selector that shows docs for a specific version.